src/AppBundle/Repository/PostRepository.php

```diff
@@ -28,6 +28,14 @@
 class PostRepository extends EntityRepository
 {
     /**
+     * @return Post[]
+     */
+    public function findAll()
```

Comment:
Probably we should use a bit different method name like `findAllOrdered()` here instead of `findAll()` as we change its normal behavior of from `EntityRepository`. What do you think?